### ISSUE TITLE
NTR: Cortisol-binding globulin low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8336]
New term request Cortisol-binding globulin low in blood (elthat_130429165331):   UMLS: 2674347

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000243
- label: Reduced circulating cortisol-binding globulin concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:11502797
- def: The concentration of cortisol-binding globulin in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8336
- synonyms:
